storybook/stories/nodes/syncing/SyncingConnecting.stories.tsx

```diff
@@ -46,21 +55,15 @@ export const DefaultSyncingConnectingStory = () => (
     onStatusIconClick={linkTo('Diagnostics', () => 'default')}
     disableDownloadLogs={boolean('disableDownloadLogs', true)}
     showNewsFeedIcon
-    blockSync={{
-      progress: 0,
-      type: 'replayedBlock',
-    }}
+    blockSync={blockSync}
```

Comment:
Could we introduce a select knob here to be able to see all 3 syncing scenarios?